app/decorators/physical_server_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class PhysicalServerDecorator < MiqDecorator
+  def fileicon
+    "svg/vendor-#{vendor.try(:downcase)}.svg"
```

Comment:
Simply adjust the tests :smile: here you should not limit yourself because of testability.